src/Microsoft.DotNet.Interactive/Kernel.Static.cs

```diff
@@ -35,19 +35,19 @@ public partial class Kernel
         /// </summary>
         /// <param name="prompt">The prompt to show.</param>
         /// <returns>The user input value.</returns>
-        public static async Task<string> GetInputAsync(string prompt = "")
+        public static async Task<string> GetInputAsync(string inputName = "", string prompt = "")
```

Comment:
This is a breaking change. Currently, we don't require an input name, so it seems correct for this to be optional, but I also think the intent of this parameter is unclear. 

Yes, it's metadata that the UI can interpret in order to provide more richer interactions if supported. Any suggestions for naming that helps make that more understandable?

Reply:
I wonder if `inputType` would be more appropriate to the use case. I think each frontend will use this to do a sort of negotiation falling back to the most appropriate UX that it supports.

Reply:
Fixed this yesterday to to have the new property be the second argument.

Reply:
@jonsequitur changed the new field to `inputType`. Any other comments?